friends/friendsModel.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require( 'mongoose' );
+
+
+const FriendsSchema = new mongoose.Schema( {
+    firstName: {
+        type: String,
+        required: true
+    },
+    lastName: {
+        type: String,
+        required: true
+    },
+    age: {
```

Comment:
Consider using a custom validator for the age ranging from 1-120
Refer to this documentation: http://mongoosejs.com/docs/validation.html#custom-validators